python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1982,6 +1982,7 @@ def renumber_vertices_by_type(self, prev_id_column=None):
             ].astype(cat_dtype)
 
         index_dtype = self.__vertex_prop_dataframe.index.dtype
+        # Should we avoid `sort_values` if we know there is only one type?
```

Comment:
The answer is yes, we shouldn't call sort_values if there is only one type.  This should match the MG behavior.

Reply:
Done.